src/main/java/mcscheduler/logic/commands/CancelLeaveCommand.java

```diff
@@ -31,7 +31,7 @@
             + "specified shift by the index numbers used in the last worker and shift listings. "
             + "\nParameters: "
             + PREFIX_SHIFT + "SHIFT_INDEX (must be a positive integer) "
-            + "[" + PREFIX_WORKER + "WORKER_INDEX (must be a positive integer)]...\n"
+            + PREFIX_WORKER + "WORKER_INDEX (must be a positive integer)...\n"
```

Comment:
`cancel leave` can take multiple workers at once, which should be signified by square brackets?
```suggestion
            + "[" + PREFIX_WORKER + "WORKER_INDEX (must be a positive integer)]...\n"
```

Reply:
From my understanding, square brackets signify a param is optional. Taking multiple workers at once is accounted for by `...`.

Reply:
ahh okay, then i think just need to change for UnassignCommand's usage message as it still uses square brackets, then can already